### DISCUSSION TITLE
Direct link video embed being cutted off

### DISCUSSION THREAD

**Discussion by illusionyy** [mmistakes/minimal-mistakes#3496] [Q&A] [answered]
### What happened?

When embedding a direct link video, it is being cutted off.

Theme version: from branch `main` as of commit 2632ff650a6efb0d856a37d675be5f1b63692181

### Expected behavior

Video should be scaled down to fit the display.

### Steps to reproduce the behavior

Embed a direct video url: https://storage.googleapis.com/assets-illusion0001/images/TLOU-Nailbomb-Softlock/T1-AC-nailbomb-section-trimmed.mp4

```html
<div align="center" class="responsive-video-container">
<video controls >
  <source src="https://storage.googleapis.com/assets-illusion0001/images/TLOU-Nailbomb-Softlock/T1-AC-nailbomb-section-trimmed.mp4" type="video/mp4">
</video>
</div>
```

Screenshot ![image](https://user-images.githubusercontent.com/37698908/155874348-dfc0c097-cf36-4266-8744-f4c85ef7444e.png)

Live site: https://illusion0001.github.io/site-testing/patches/2021/12/04/t1-nailbomb-softlock-patch/

### Other

Site repository: https://github.com/illusion0001/site-testing

**Comment by mmistakes** [comment, marked as answer]
`.responsive-video-container` has styling to make `iframe` video content (e.g., YouTube, Vimeo, etc.) responsively size.

You don't need it for a `<video>` element. Just add `width="100%"` and it'll work.

```html
<video controls width="100%">
  <source src="https://storage.googleapis.com/assets-illusion0001/images/TLOU-Nailbomb-Softlock/T1-AC-nailbomb-section-trimmed.mp4" type="video/mp4">
</video>
```

![image](https://user-images.githubusercontent.com/1376749/155999940-08a706b1-81f8-4dc8-bb37-73760a4af5ed.png)

Ref: https://css-tricks.com/fluid-width-video/

**Reply by illusionyy** [reply]
Thank you!
